docs/pages/index.tsx

```diff
@@ -1,8 +1,12 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
+import { useRouter } from 'next/router';
+import Link from 'next/link'
 import styles from '../styles/Home.module.css';
 
 const Home: NextPage = () => {
+  const router = useRouter();
```

Comment:
why does this need to be here?

Reply:
This is how nextJS does routes. The router is and object that handles navigation

https://nextjs.org/docs/api-reference/next/router#router-object

Reply:
I can remove this since we're not using it.